planner_node:
  agent:
    mode: "planner" # benchmark / ablation
    planner: "moma" # ddim, moma
    local_mode: false
    planning_budget: 0.5
    replan_interval: 100.0
    planning_horizon: 6000.0
    only_front: false
    fixed_startgoal: false
    random_ee: true
    pick_state: [3.3, 0.0, 0.0, 1.0259568691253662, 0.7873354554176331, -0.037315141409635544, 0.024556782096624374, 0.09346238523721695, 0.7401068210601807, -1.14198637008667]
    mid_state: [9.0, 1.5, 0.0, 1.0259568691253662, 0.7873354554176331, -0.037315141409635544, 0.024556782096624374, 0.09346238523721695, 0.7401068210601807, -1.14198637008667]
    place_state: [11.9, -0.3, 0.0, 1.0259568691253662, 0.7873354554176331, -0.037315141409635544, 0.024556782096624374, 0.09346238523721695, 0.7401068210601807, -1.14198637008667]
    fixed_sequence: false
    startgoal_dist_range : [3.0, 8.0]
    stat_num: 1000
    scene: "tables" # cuboids / tables
